src/search_recipe/QueryAPI.java

```diff
@@ -15,10 +15,10 @@ public class QueryAPI {
     /*
     This calls the API with the passed in arraylist of ingredients and the number of results you want to return
      */
-    public static JsonArray getResults(ArrayList<String> ingredients, int number) throws Exception {
+    public static JsonArray getResults(ArrayList<String> ingredients, int number) {
```

Comment:
Is it possible for the API to throw exceptions? If Yes then how we are going to deal with this case?